ports/nrf/mpconfigport.mk

```diff
@@ -52,6 +52,8 @@ MCU_SUB_VARIANT = nrf52840
 # Fits on nrf52840 but space is tight on nrf52833.
 CIRCUITPY_AESIO ?= 1
 
+CIRCUITPY_MSGPACK ?= 1
```

Comment:
I suggested doing this in `py` based on full build. That should enable it on most builds including nrf by default.

Reply:
Done